extensions/wikia/AchievementsII/services/AchImageUploadService.class.php

```diff
@@ -167,43 +167,47 @@ public static function uploadBadge($destinationFileName, $badgeLevel) {
 		imagedestroy($badgeImage);
 		$badgeImage = $tmp;
 
-		// load images
-		$badgesFragments = dirname(__FILE__) . '/../images/badges/fragments';
+		if ($wgBadgeCustomizationAllowed) {
```

Comment:
I think this variable name is confusing. No matter if it's true or false we allow for badge customization. This change only if we apply our overlay on top of it or not.